homework12/task/migration/migrations/0002_auto_20210819_2108.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.6 on 2021-08-19 21:08
+import datetime
+
+from django.db import migrations
+
+
+def add_row(apps, schema_editor):
+    Teacher = apps.get_model("migration", "Teacher")
+    t = Teacher(first_name="Ivan", last_name="Wonderful")
+    t.save()
+
+    Student = apps.get_model("migration", "Student")
+    st = Student(first_name="Olya", last_name="Yalo")
+    st.save()
+
+    Homework = apps.get_model("migration", "Homework")
+    hw = Homework(
+        text="Calculate integral", deadline=datetime.timedelta(days=7), teacher_id=2
+    )
+    hw.save()
+
+    HomeworkResult = apps.get_model("migration", "HomeworkResult")
+    hw_result = HomeworkResult(hw_id=2, solution="Done", student_id=2)
+    hw_result.save()
```

Comment:
В ORM так не надо использовать сырые айтишники, обычно внешняя сущность указывается буквально, а не по айдишнику. В твоём случае код миграции надо делать так:
```suggestion
    Homework = apps.get_model("migration", "Homework")
    hw = Homework(
        text="Calculate integral", deadline=datetime.timedelta(days=7), teacher=t
    )
    hw.save()

    HomeworkResult = apps.get_model("migration", "HomeworkResult")
    hw_result = HomeworkResult(hw=hw, solution="Done", student=st)
    hw_result.save()
```